include/api/mli_krn_avepool_spec_api.h

```diff
@@ -26,84 +26,64 @@ extern "C" {
 
 mli_status mli_krn_avepool_chw_fx16_k2x2_str1_nopad(const mli_tensor * in, const mli_pool_cfg * cfg, mli_tensor * out);
```

Comment:
This onde (and next) are of str_1 assumption. But I can't find mli_krn_avepool_chw_fx16_k2x2_nopad

Reply:
The same for fx8

Reply:
I have added it in the next commit